### DISCUSSION TITLE
Add an option to set lazer use stable directory structure

### DISCUSSION THREAD

**Discussion by HollisMeynell** [ppy/osu#30983] [Ideas]
I have a large disk, enough to hold all of osu beatmaps, so duplicate files aren't a concern for me. It would be great to have the option to turn off file management instead of it being mandatory. 

Currently, the file management system makes it difficult to directly find and edit files. Personally, I prefer using software like 'Everything' for file searches or tools to monitor score modifications (mapping tools). However, the current setup in Lazer makes these workflows either unusable or overly complicated. I’d really appreciate an option to disable file management and work with a stable directory structure instead.

Orz

**Comment by peppy** [comment]
Which mapping tools specifically are you talking about here? We might be able to provide alternative recommendations.

That said, can't / won't be giving this option, so if you're not willing to find compromises, then stable is your best bet.
